rrule/Cargo.toml

```diff
@@ -43,4 +47,11 @@ cli-tool = ["clap"]
 serde = ["serde_with", "chrono/serde", "chrono-tz/serde"]
 
 # Allows EXRULE's to be used in the `RRuleSet`.
-exrule = []
\ No newline at end of file
+exrule = []
+
+# Allows to use WASM
+wasm = []
+
+[lib]
+#[cfg(feature = "wasm")]
```

Comment:
`#[cfg(feature = "wasm")]` does nothing in the toml file as it is just a comment. Please take a look at `serde` and `cli-tool` features for how to only pull in certain dependencies when a feature is enabled.